package.json

```
@@ -189,7 +189,7 @@
     "typescript": "4.6.3"
   },
   "engines": {
-    "node": "^18 || ^20"
+    "node": "^18 || ^20 || >=21"
```

```suggestion
    "node": ">=20"
```
I meant this, we can change node version to 21, but on most of the products migrated to pnpm I set >=20

---

I just reverted the changes, I think we better update node version as a separate scope.